### PR TITLE
Make early stage of Lawnchair 15 launchable in A14

### PR DESCRIPTION
**Pull request by validcube**
## Description

Make Lawnchair 15 launchable in the early stage, tested with A14

This PR is the same one used at #4973

## Type of change

:x: General change (non-breaking change that doesn't fit the below categories like copyediting)
✅ Bug fix (non-breaking change which fixes an issue)
:x: New feature (non-breaking change which adds functionality)
:x: Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: flags/src/com/android/launcher3/FeatureFlagsImpl.java [hunk 1]
```diff
@@ -141,14 +141,7 @@ private void load_overrides_launcher() {
...
         } catch (NullPointerException e) {
-            throw new RuntimeException(
-                    "Cannot read value from namespace launcher "
-                            + "from DeviceConfig. It could be that the code using flag "
-                            + "executed before SettingsProvider initialization. Please use "
-                            + "fixed read-only flag by adding is_fixed_read_only: true in "
-                            + "flag declaration.",
-                    e
-            );
+            // Ignored
```
Comment (CoelacanthusHex) [inline flags/src/com/android/launcher3/FeatureFlagsImpl.java:144]: Would it be better to use a tagged comment like `FIXME:` or `TODO:` to find them quickly later?

**Comment by BlueGradientHorizon** [comment]
Hello. It fixes an exception about setEnableOnBackInvokedCallback for me, but LC still crashes: https://paste.crdroid.net/QQJEWx

**Comment by BlueGradientHorizon** [comment]
After this commit LC starts successfully on POCO F5, HyperOS 1.0.6.0.UMRMIXM (A14), but crashes with exact same reason on POCO X3 Pro, crDroid 11.0-beta, Nov. 4 build (A15, AP3A.241005.015)

**Review by MrSluffy — APPROVED** [review]
LGTM! Thanks for contributing!

File: flags/src/com/android/launcher3/FeatureFlagsImpl.java (hunk 1 above)
Comment (validcube) [reply flags/src/com/android/launcher3/FeatureFlagsImpl.java:144]: 👋 @CoelacanthusHex (this message was likely a day ago, but I just never hit send again!)

This is an idea that I thought of, but for now, I see no reason that feature flags should be fully initialised at the moment as feature flags are going to be the most obscure feature of Lawnchair/Launcher3. But I do agree that we would likely have to find them later. Perhaps making an error log every time there's a violation would be nicer than ignoring the problem entirely.

(Note: I might be wrong, I don't have a phone to test my claim now that the screen is cracked out of existence and emulator is too slow for my computer to handle)

**Comment by validcube** [comment]
> After this commit LC starts successfully on POCO F5, HyperOS 1.0.6.0.UMRMIXM (A14), but crashes with exact same reason on POCO X3 Pro, crDroid 11.0-beta, Nov. 4 build (A15, AP3A.241005.015)

👋 @BlueGradientHorizon (this message was likely a day ago, but I just never hit send again again!)

Please file an issue in Lawnchair.

**Pull request merged**